client/ayon_core/pipeline/load/plugins.py

```diff
@@ -147,10 +151,20 @@ def is_compatible_loader(cls, context):
         if "*" in plugin_product_types:
             return True
 
+        plugin_product_base_types = set(plugin_product_base_types)
```

Comment:
This whole filtering logic is not right, with more filters the logic has to be reverted.
```python
if (
    "*" not in plugin_product_types
    or product_type not in plugin_product_types
):
    return False

if (
    "*" not in plugin_product_base_types
    or product_base_type not in plugin_product_base_types
):
    return False
return True
```